projects/nae-example-app/src/app/doc/public-task-view/public-task-view.component.ts

```diff
@@ -112,8 +116,13 @@ export class PublicTaskViewComponent extends AbstractTaskView implements AfterVi
 
     @ViewChild('header') public taskHeaderComponent: HeaderComponent;
 
-    constructor(taskViewService: TaskViewService) {
+    constructor(taskViewService: TaskViewService, publicTaskLoadingService: PublicTaskLoadingService) {
```

Comment:
Component?

Reply:
The public task view does not have a separate component in the engine. So this implementation is within the constructor only in the example app. This part must be copied and pasted into the public task view in the project for proper functionality.